python/ola/OlaClient.py

```diff
@@ -179,25 +189,30 @@ def __repr__(self):
                     nr_outputs=len(self.output_ports))
 
   def __lt__(self, other):
+    if other.__class__ is not self.__class__:
+      return NotImplemented
     return self.alias < other.alias
 
   def __eq__(self, other):
+    if other.__class__ is not self.__class__:
+      return False
     return self.alias == other.alias
 
-  # These 3 could be replaced by functools.total_ordering when support
-  # for 2.6 is dropped.
   def __le__(self, other):
+    if other.__class__ is not self.__class__:
+      return NotImplemented
     return self.alias <= other.alias
 
   def __gt__(self, other):
+    if other.__class__ is not self.__class__:
+      return NotImplemented
     return self.alias > other.alias
 
   def __ge__(self, other):
+    if other.__class__ is not self.__class__:
+      return NotImplemented
     return self.alias >= other.alias
 
-  def __ne__(self, other):
```

Comment:
How come this has been dropped?

Reply:
Inattention to detail. In python 3 its value is `not __eq__` by default, but it's not in python 2. My bad.